Include/pyport.h

```diff
@@ -572,6 +572,9 @@ extern "C" {
 #  if defined(__SANITIZE_ADDRESS__)
 #    define _Py_ADDRESS_SANITIZER
 #  endif
+#  if defined(__SANITIZE_THREAD__)
```

Comment:
This code path is only entered when `defined(__has_feature)` is false, no?

Reply:
Yeah, it looks so. Do you have any concerns?

Reply:
Ahah. Yes, you're right, nevermind.